src/constraints/scalaraffine.jl

```diff
@@ -169,6 +169,24 @@ function MOI.modify!(m::LinQuadOptimizer, c::LCI{S}, chg::MOI.ScalarCoefficientC
     change_matrix_coefficient!(m, m[c], col, chg.new_coefficient)
 end
 
+MOI.canset(m::LinQuadOptimizer, ::MOI.ConstraintFunction, ::Type{LCI{S}}) where {S <: Union{LE, GE, EQ}} = true
+function MOI.set!(m::LinQuadOptimizer, attr::MOI.ConstraintFunction, CI::LCI{S}, replacement::Linear) where {S <: Union{LE, GE, EQ}}
+    previous = MOI.get(m, attr, CI)
+    for term in previous.terms
+        var = term.variable_index
+        chg = MOI.ScalarCoefficientChange{Float64}(var, zero(Float64))
+        MOI.modify!(m, CI, chg)
+    end
+    for term in replacement.terms
+        var = term.variable_index
+        chg = MOI.ScalarCoefficientChange{Float64}(var, term.coefficient)
+        MOI.modify!(m, CI, chg)
+    end
+    change_rhs_coefficient!(m, m[CI], -replacement.constant)
```

Comment:
Does this handle the constant encoded in the set properly (e.g. `MOI.EqualTo(1.0)`)?